### PR TITLE
chore(connect-common): add fw 2.8.6. for t3t1

### PR DESCRIPTION
**Pull request by MiroslavProchazka**
Adding FW binaries for upcoming 2.8.6 FW release

### CODE REVIEW

File: packages/connect-common/files/firmware/t3t1/releases.json [hunk 1]
```diff
@@ -1,4 +1,20 @@
 [
+    {
+        "required": false,
+        "version": [2, 8, 6],
+        "min_bridge_version": [2, 0, 7],
```
Comment (komret) [inline packages/connect-common/files/firmware/t3t1/releases.json]: Suite doesn't use this property, should be removed.

File: packages/connect-common/files/firmware/t3t1/releases.json [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
+        "min_bootloader_version": [2, 1, 6],
+        "min_firmware_version": [2, 7, 2],
+        "bootloader_version": [2, 1, 9],
+        "firmware_revision": "17f2a3256b2631823c2bad1855fab61b27985fd2",
+        "translations": ["cs-CZ", "de-DE", "es-ES", "fr-FR", "it-IT", "pt-BR"],
+        "url": "data/firmware/t3t1/trezor-t3t1-2.8.6.bin",
```
Comment (komret) [inline packages/connect-common/files/firmware/t3t1/releases.json]: The `data/` part of the path is only relevant for the data repo, should be removed.

File: packages/connect-common/files/firmware/t3t1/releases.json [hunk 3]
```diff
@@ -1,4 +1,20 @@
...
+        "fingerprint": "da8741ccfb4a328e639351dfa5dba496198016bdbd076ec02a40a52870d85e91",
+        "changelog": "* Show last typed PIN number for short period of time.\n* Add P2WSH support for Unchained BIP32 paths.\n* Simplified UI of Cardano transactions initiated by Trezor Suite.\n* Improve UI synchronization, ordering, and responsiveness.\n* Improve device responsiveness by removing unnecessary screen refreshes.\n* Bootloader 2.1.9. included\n* Show account info in ETH send/stake flow.\n* Fix ETH account number detection.\n* Fix XPUB confirmed success screen title.\n* New EVM call contract flow UI.\n* Fix translation of the 'Enable labeling' screen.",
+        "url_bitcoinonly": "data/firmware/t3t1/trezor-t3t1-2.8.6-bitcoinonly.bin",
```
Comment (komret) [inline packages/connect-common/files/firmware/t3t1/releases.json]: As above.

**Review by komret — CHANGES_REQUESTED** [review]

**Review by komret — APPROVED** [review]

**Comment by bosomt** [comment]
**QA OK**

**Info:**
 - Suite version: desktop 24.12.1 (ec8321d0d17529ca77d8916e70158cecc469f863)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/24.12.1 Chrome/128.0.6613.186 Electron/32.2.6 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T3T1 2.8.6 regular (revision 17f2a3256b2631823c2bad1855fab61b27985fd2)
 - Transport: BridgeTransport 2.0.33

**Pull request merged**